app.py

```
@@ -7,38 +7,53 @@
+        # Alle Werte aus der DEFAULT Section extrahieren
+        for key, value in config_parser['DEFAULT'].items():
+            config[key.upper()] = value
+        
+        logger.info(f"Konfiguration erfolgreich geladen aus {config_file}")
         return config
+        
     except FileNotFoundError:
-        print(f"Fehler: Konfigurationsdatei {config_file} nicht gefunden")
-        return {}
+        logger.error(f"Konfigurationsdatei {config_file} nicht gefunden")
+        raise FileNotFoundError(f"Konfigurationsdatei {config_file} nicht gefunden")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)